### DISCUSSION TITLE
Native Hook doesn't work for pure arm app on x86 device

### DISCUSSION THREAD

**Discussion by ElysiaEgo** [LSPosed/LSPosed#2145] [Q&A] [answered]
It seems that NativeBridge doesn't use do_dlopen (I didn't find it being called in the android src).
Maybe we can hook [NativeBridgeLoadLibrary](https://cs.android.com/android/platform/superproject/+/master:art/libnativebridge/native_bridge.cc;l=563?q=NativeBridgeLoadLibrary&sq=&hl=zh-cn) to get the handle.

**Comment by yujincheng08** [comment, marked as answer]
It wont work. Dobby cannot hook crossarchly.
